Makefile

```diff
@@ -0,0 +1,68 @@
+artifact_name       := customer-feedback-api
+version             := "unversioned"
+
+.PHONY: clean
+clean:
+	@echo "Running clean"
+	mvn clean
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+	rm -f ./build.log
+	@echo "Finished clean"
+
+.PHONY: build
+build:
+	@echo "Running build"
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	mvn package -DskipTests=true
+	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
+	@echo "Finished build"
+
+.PHONY: test
+test: test-unit
+
+.PHONY: test-unit
+test-unit: clean
+	mvn test
+
+.PHONY: package
+package:
+ifndef version
+	$(error No version given. Aborting)
+endif
+	$(info Packaging version: $(version))
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	mvn package -DskipTests=true
+	$(eval tmpdir:=$(shell mktemp -d build-XXXXXXXXXX))
+	cp ./start.sh $(tmpdir)
+	cp ./routes.yaml $(tmpdir)
+	cp ./target/$(artifact_name)-$(version).jar $(tmpdir)/$(artifact_name).jar
+	cd $(tmpdir); zip -r ../$(artifact_name)-$(version).zip *
+	rm -rf $(tmpdir)
+
+.PHONY: dist
+dist: clean build package
+
+.PHONY: sonar
+sonar:
+	mvn sonar:sonar
+
+.PHONY: sonar-pr-analysis
+sonar-pr-analysis:
+	mvn sonar:sonar -P sonar-pr-analysis
+
+.PHONY: dependency-check
```

Comment:
The security-check task in concourse relies on this job being named security-check - https://github.com/companieshouse/ci-concourse-resources/blob/shared-services/tasks/java/21/corretto/security-check/task.yml#L21